README.md

```diff
@@ -1,3 +1,8 @@
+[![Hackage version](https://img.shields.io/hackage/v/hpack.svg?label=Hackage)](http://hackage.haskell.org/package/hpack)
+[![Stackage LTS version](https://www.stackage.org/package/hpack/badge/lts?label=Stackage)](https://www.stackage.org/package/hpack)
+[![hpack on Stackage Nightly](https://stackage.org/package/hpack/badge/nightly)](https://stackage.org/nightly/package/hpack)
+[![Travis build status](https://travis-ci.org/sol/hpack.svg?branch=master)](https://travis-ci.org/sol/hpack)
```

Comment:
This needs updating.  I think you had that in your other PR.  Can you migrate it over?

Reply:
Sigh.  Ok.  (These things are easier for committers than for the unprivileged folks...)

Reply:
We don't run travis builds anymore.  So I think we want to remove this badge altogether.  Just the GitHub Actions one is fine.

Reply:
I removed the travis badge.